docs/devGuide/development/workflow.md

```diff
@@ -190,6 +190,20 @@ After which, you can update the **expected** test files with: `npm run updatetes
   as uncommitted changes due to the way they are generated. If you are not directly modifying those files in your PR, you should **discard those changes** and **do not commit** them.
 </box>
 
+<box type="tip" seamless header="Solving merge conflicts in expected test files">
+
+1. Ensure the fork is latest
+   - Pull from upstream repo to fork
+2. Ensure local master is in sync with fork master
+   - Pull from the fork master into local master
```

Comment:
I think this overlaps with the first step!

Reply:
I think what the first step is saying is remote fork is synced with upstream repo. Second step is saying that local master is in sync with remote fork :)